tests/python/unittest/test_loss.py

```diff
@@ -348,6 +348,22 @@ def test_triplet_loss():
             optimizer='adam')
     assert mod.score(data_iter, eval_metric=mx.metric.Loss())[0][1] < 0.05
 
+@with_seed()
```

Comment:
Please add additional tests for label = -1 and hybridization.

Reply:
> Please add additional tests for label = -1 and hybridization.

1. Added tests for label 1 and -1 in a randomly generated set of labels,
2. This function is meant to be utility similar to cosine_distance.